pkg/controller/podautoscaler/metrics/metrics_client.go

```diff
@@ -44,11 +44,11 @@ type MetricsClient interface {
 	// GetCPUUtilization returns the average utilization over all pods represented as a percent of requested CPU
 	// (e.g. 70 means that an average pod uses 70% of the requested CPU)
 	// and the time of generation of the oldest of utilization reports for pods.
-	GetCPUUtilization(namespace string, selector map[string]string) (*int, time.Time, error)
+	GetCPUUtilization(namespace string, selector string) (*int, time.Time, error)
```

Comment:
Yes. That allows for things like making a pass-through selector type that doesn't have to understand the selector itself.